paparazzi/src/main/java/app/cash/paparazzi/HtmlReportWriter.kt

```diff
@@ -88,20 +97,47 @@ internal class HtmlReportWriter(
       }
 
       override fun close() {
-        if (hashes.size == 1) {
-          shots += snapshot.copy(file = "images/${hashes[0]}.png")
+        val shot = if (hashes.size == 1) {
+          val original = File(imagesDirectory, "${hashes[0]}.png")
+          if (isRecording) {
```

Comment:
i'm back and forth about whether to extract this out of HtmlReportWriter and into a, say, SnapshotRecorder, mainly because there'd be a bit of duplication.

See here for previous attempt: https://github.com/cashapp/paparazzi/pull/69/files#r326872374